control/bgp.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: 3.0.3
+info:
+  title: BGP Control models
+  description: >-
+    All BGP control schemas
+  version: ^0.0.0
+components:
+  schemas:
+    BGP.Notification:
+      description: >-
+        A NOTIFICATION message is sent when an error is detected with the BGP session, such as hold timer expiring, misconfigured AS number 
+        or a BGP session reset is requested. This causes the BGP connection to close. Send explicit NOTIFICATIONs for list of specified 
+        BGP peers. If a user wants to send custom Error Code and Error Subcode the custom object should be used.
```

Comment:
If a user wants to send custom Error Code and Error Subcode the custom object should be ..."used" "configured"?

Reply:
updated